ApexOSA/filter.py

```
@@ -0,0 +1,271 @@
+        if isinstance(Type, str):
+            if Type.lower() == "repeat":
+                Type = 2
+            else:
+                Type = 1
+        elif isinstance(Type, int):
+            if Type == 2:
+                Type = 2
+            else:
+                Type = 1
+        else:
+            raise ApexError(APXXXX_ERROR_ARGUMENT_TYPE, "Type")
```

Ruff (0.12.2)

`ApexError` may be undefined, or defined from star imports

(F405)